learningresources/models.py

```diff
@@ -88,5 +92,19 @@ def save(self, *args, **kwargs):
                 slug = "{0}{1}".format(slugify(self.name), count)
                 count += 1
             self.slug = slug
+            is_create = True
+        new_repository = super(Repository, self).save(*args, **kwargs)
+        if is_create:
+            roles_init_new_repo(self)
+        return new_repository
 
-        return super(Repository, self).save(*args, **kwargs)
+    class Meta:
+        # pylint: disable=missing-docstring
+        # pylint: disable=old-style-class
+        # pylint: disable=no-init
+        # pylint: disable=too-few-public-methods
+        permissions = (
+            RepoPermission.edit_repo,
+            RepoPermission.use_repo,
```

Comment:
I wanted to distinguish the authorization given to the Curator and the one given to the Author.
Initially I did not have a clear idea of how the urls and the user experience would look like. Probably now we do not need 2 different permissions.